### DISCUSSION TITLE
Adding a symbol makes the PDF huge

### DISCUSSION THREAD

**Discussion by adrfantini** [typst/typst#4078] [General]
### Description

I'm compiling on ArchLinux, and using any `sym` symbol increases the PDF size by about 9.7MB.
I cannot reproduce it on the web app (see below link), but with the same code, it can be reliably reproduced on my laptop.

Is it a matter of fonts (I'm using Noto Fonts by default)? How to fix it?

If it's a font issue, maybe a compiler warning could be useful?

Thanks!

### Reproduction URL

https://typst.app/project/rnjf_mj61DhePSKMgYy1aa

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by PgBiel** [comment]
Related: https://github.com/typst/typst/issues/895

**Comment by PgBiel** [comment]
> I cannot reproduce it on the web app (see below link), but with the same code, it can be reliably reproduced on my laptop.

Which Typst version are you using? (From `typst --version`)

**Reply by adrfantini** [reply]
I'm using what I believe is the latest official version, typst 0.11.0 (2bf9f95d)

**Comment by Enivex** [comment]
It's one of two things. Either the subsetting @PgBiel mentioned, or because those symbols can become emojis depending on the font used.

Unless you explicitly set an emoji font, the font used will vary between your computer and the web-app depending on the fallback font used. Moreover, emojis are broken in pdf in the current release (they work on main).

**Comment by LaurenzV** [comment]
It is most likely https://github.com/typst/typst/issues/3777 it should be fixed on main.

**Reply by adrfantini** [reply]
That's awesome, thanks!

Any idea when it'll be out, or how I can work around it? Simply using a different font should be enough? Can I specify a font specifically for emojis?

**Reply by LaurenzV** [reply]
0.12 is probably still a while away. So the workarounds are
1) You compile Typst yourself and compile your document with that.
2) You can try using https://github.com/polazarus/typst-svg-emoji

Using another font won't be of much help because no emoji fonts work on Typst 0.11.

**Reply by adrfantini** [reply]
thank you, appreciate it
